packages/node/src/lib/notificationGroups/notificationGroups.ts

```diff
@@ -0,0 +1,18 @@
+import { WithHttp } from '../novu.interface';
+import {
+  INotificationGroups,
+  INotificationGroupsPayloadOptions,
+} from './notificationGroups.interface';
+
+export class NotificationGroups
+  extends WithHttp
+  implements INotificationGroups
+{
+  async get() {
+    return await this.http.get(`/notification-groups`);
+  }
+
+  async post(data: INotificationGroupsPayloadOptions) {
```

Comment:
```suggestion
  async create(name: string) {
```

Reply:
Good with this if the expectation is to have a `(primary argument, data)` format. Changing